x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/policy_template_form.tsx

```diff
@@ -447,7 +450,9 @@ const AzureAccountTypeSelect = ({
             type: 'text',
           },
           'azure.credentials.type': {
-            value: AZURE_ARM_TEMPLATE_CREDENTIAL_TYPE,
+            value: isAgentless
+              ? 'service_principal_with_client_secret'
```

Comment:
I plan to move this into constants together with GCP credential types in the follow up PR